docs/hooks/appointment-book.md

```diff
@@ -8,7 +8,7 @@
 
 ## Workflow
 
-This hook is invoked when the user is scheduling one or more future encounters/visits for the patient.  It may be invoked at the start and end of the booking process and/or any time between those two points.  It allows CDS Services to intervene in the decision of when future appointments should be scheduled, where they should be scheduled, what services should be booked, to identify actions that need to occur prior to scheduled appointments, etc.
+This hook is invoked when the user is scheduling one or more future encounters/visits for the patient.  It may be triggered for an appointment with the appointment creator, a clinician within the same organization as the appointment creator or even for an appointment outside the creator's organization.  It may be invoked at the start and end of the booking process and/or any time between those two points.  This hook enables CDS Services to intervene in the decision of when future appointments should be scheduled, where they should be scheduled, what services should be booked, to identify actions that need to occur prior to scheduled appointments, etc.  
```

Comment:
> It may be triggered for an appointment with the appointment creator, a clinician within the same organization as the appointment creator or even for an appointment outside the creator's organization.

Is the intention here to constrain the types of appointments for which this can be triggered or merely to give examples?  If it's the latter, can we make that clearer?  "Examples include an appointment with..." or something along those lines?

Reply:
@dennispatterson - updated, re-review?